packages/components/src/NotConnected/NotConnected.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import propTypes from 'prop-types';
+import { EmptyState, EmptyStateIcon, EmptyStateBody, Title, Button } from '@patternfly/react-core';
+import { DisconnectedIcon } from '@patternfly/react-icons';
+
+const NotConnected = ({ titleText, bodyText, buttonText }) => {
+    return (
+        <EmptyState>
+            <EmptyStateIcon icon={DisconnectedIcon} />
+            <Title headingLevel="h5" size="lg">
+                {titleText}
+            </Title>
+            <EmptyStateBody>
+                {bodyText}
+            </EmptyStateBody>
+            <Button
+                variant="primary"
+                component="a"
+                href="http://access.redhat.com/products/cloud_management_services_for_rhel#getstarted"
+                className="pf-u-mt-lg"
+            >
+                {buttonText}
+            </Button>
+        </EmptyState>
+    );
+};
+
+NotConnected.propTypes = {
+    titleText: propTypes.string,
+    bodyText: propTypes.string,
+    buttonText: propTypes.string
```

Comment:
I think we should change this to `node` to enable elements too (for example, translatable texts)